### PR TITLE
Update docs for pluggin in CA certs

### PR DESCRIPTION
**Pull request by nmittler**
The new recommended approach is to generate the certs and to use intermediate certs for each cluster.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 1]
```diff
@@ -1,54 +1,91 @@
...
+description: Shows how system administrators can configure Istio's CA with a root certificate, signing certificate and key.
 weight: 80
 keywords: [security,certificates]
 aliases:
     - /docs/tasks/security/plugin-ca-cert/
 test: true
 ---
 
-This task shows how administrators can configure the Istio certificate authority with an existing root certificate, signing certificate and key.
+This task shows how administrators can configure the Istio certificate authority with a root certificate,
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:11]:
```suggestion
This task shows how administrators can configure the Istio certificate authority (CA) with a root certificate,
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 2]
```diff
@@ -1,54 +1,91 @@
...
 
 By default, Istio's CA generates a self-signed root certificate and key, and uses them to sign the workload certificates.
 Istio's CA can also sign workload certificates using an administrator-specified certificate and key, and with an
-administrator-specified root certificate. This task demonstrates how to plug such certificates and key into Istio's CA.
+administrator-specified root certificate.
 
-## Plugging in existing certificates and key
+The root CA is used by all workloads within the mesh as the root of trust. Each Istio CA uses an intermediate CA
+signing key and certificate, signed by the root CA. When multiple Istio CAs exist within the mesh, this establishes a
+hierarchy of trust among the CAs.
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:20]:
```suggestion
A root CA is used by all workloads within a mesh as the root of trust. Each Istio CA uses an intermediate CA
signing key and certificate, signed by the root CA. When multiple Istio CAs exist within a mesh, this establishes a
hierarchy of trust among the CAs.
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 3]
```diff
@@ -1,54 +1,91 @@
...
 
-Suppose we want to have Istio's CA use an existing signing (CA) certificate `ca-cert.pem` and key `ca-key.pem`.
-Furthermore, the certificate `ca-cert.pem` is signed by the root certificate `root-cert.pem`.
-We would like to use `root-cert.pem` as the root certificate for Istio workloads.
+{{< image width="80%"
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:22]: Diagram is too small and surrounded by whitespace. See preview.

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 4]
```diff
@@ -1,54 +1,91 @@
...
+    link="hierarchical-pki.png"
+    caption="Hierarchical PKI"
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:24]: I wouldn't use acronym `PKI` in the caption, unless it appears in (and is expanded) in the body text as well.

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 5]
```diff
@@ -1,54 +1,91 @@
...
+    >}}
 
-In the following example,
-Istio CA's signing (CA) certificate (`ca-cert.pem`) is different from the root certificate (`root-cert.pem`),
-so the workload cannot validate the workload certificates directly from the root certificate.
-The workload needs a `cert-chain.pem` file to specify the chain of trust,
-which should include the certificates of all the intermediate CAs between the workloads and the root CA.
-In our example, it contains Istio CA's signing certificate, so `cert-chain.pem` is the same as `ca-cert.pem`.
-Note that if your `ca-cert.pem` is the same as `root-cert.pem`, the `cert-chain.pem` file should be empty.
+This task demonstrates how to generate and plug in certificates and key into Istio's CA. These steps can be repeated
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:27]:
```suggestion
This task demonstrates how to generate and plug in the certificates and key for Istio's CA. These steps can be repeated
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 6]
```diff
@@ -1,54 +1,91 @@
...
-These files are ready to use in the `samples/certs/` directory.
+## Plug in certificates and key into the cluster
 
 {{< tip >}}
-The default Istio CA installation configures the location of certificates and keys based on the
-predefined secret and file names used in the command below (i.e., secret named `cacerts`, root certificate
-in a file named `root-cert.pem`, Istio CA's key in `ca-key.pem`, etc.).
-You must use these specific secret and file names, or reconfigure Istio's CA when you deploy Istio.
+For production cluster setup, it is a good practice to do the following on an offline machine with good
+security protection. The root private key should be exposed to minimal people and processes.
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:34]:
```suggestion
security protection. The root private key should be exposed to as few people and processes as possible.
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 7]
```diff
@@ -123,12 +160,18 @@ openssl command is expected.
...
 
     {{< text bash >}}
-    $ openssl verify -CAfile <(cat samples/certs/ca-cert.pem samples/certs/root-cert.pem) ./proxy-cert-1.pem
+    $ openssl verify -CAfile <(cat samples/certs/cluster1/ca-cert.pem samples/certs/cluster1/root-cert.pem) ./proxy-cert-1.pem
     ./proxy-cert-1.pem: OK
     {{< /text >}}
 
 ## Cleanup
 
+*   To remove the certificates, keys, and intermediate files from your local disk:
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:169]:
```suggestion
*   Remove the certificates, keys, and intermediate files from your local disk:
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 8]
```diff
@@ -123,12 +160,18 @@ openssl command is expected.
...
+
+    {{< text bash >}}
+    $ rm -rf cluster1
+    {{< /text >}}
+
 *   To remove the secret `cacerts`, and the `foo` and `istio-system` namespaces:
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:175]:
```suggestion
*   Remove the secret `cacerts`, and the `foo` and `istio-system` namespaces:
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 9]
```diff
@@ -1,54 +1,91 @@
 ---
-title: Plugging in existing CA Certificates
-description: Shows how system administrators can configure Istio's CA with an existing root certificate, signing certificate and key.
+title: Plug in CA Certificates
+description: Shows how system administrators can configure Istio's CA with a root certificate, signing certificate and key.
 weight: 80
 keywords: [security,certificates]
 aliases:
     - /docs/tasks/security/plugin-ca-cert/
 test: true
```
Comment (frankbu) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:8]: Test needs to be updated.

Comment (nmittler) [reply content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md]: @frankbu there is no test currently ... do you mean snips.sh? It's updated now.

Comment (nmittler) [reply content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md]: oh nm ... looks like it was just checked in.  I see it now ... will update.

**Comment by ericvn** [comment]
Looking at some of the k8s test errors, these match what I am trying to fix in #8103 in moving to the `master` branch which is also being done here. I hope to have them fix relatively soon.

**Comment by ericvn** [comment]
LGTM, but wiill need to sync the Makefile across.

**Review by frankbu — APPROVED** [review]

File: bin/init.sh [hunk 10]
```diff
@@ -62,8 +62,9 @@ cp -a "${ISTIO_OUT}/release/istioctl-linux-amd64" /gobin/istioctl
 popd > /dev/null
 
 # Copy install/samples files over from Istio. These are needed by the tests.
-rm -rf "${ISTIOIO_GO}/samples" "${ISTIOIO_GO}/tests/integration" "${ISTIOIO_GO}/manifests"
+rm -rf "${ISTIOIO_GO}/samples" "${ISTIOIO_GO}/tools" "${ISTIOIO_GO}/tests/integration" "${ISTIOIO_GO}/manifests"
 cp -a "${ISTIO_GO}/samples" "${ISTIOIO_GO}/samples"
+cp -a "${ISTIO_GO}/tools" "${ISTIOIO_GO}/tools"
```
Comment (su225) [inline bin/init.sh:67]: This is pulling a lot of files. (~60) - even [perhaps] unrelated ones like istio-iptables, packaging, bug-report etc! Is this needed or should this be narrowed down to `${ISTIO_GO}/tools/certs`?

**Pull request merged**